src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -513,6 +514,10 @@ private static void ConfigureAuthentication(WebApplicationBuilder builder, Dashb
 
                     // Avoid "message.State is null or empty" due to use of CallbackPath above.
                     options.SkipUnrecognizedRequests = true;
+
+                    // Allow the requirement of HTTPS communication with the OpenIdConnect authority to be
+                    // relaxed, for unit testing purposes only.
+                    options.RequireHttpsMetadata = requireHttpsMetadataForOpenIdConnect;
```

Comment:
Does this prevent the user from being able to configure this in appSettings? I guess theoretically that applies to anything above as well. I just noticed this one in particular because it was something I had to set in config when testing OIDC with keycloak

Reply:
I guess a follow-up question would then be, can this be set via config (e.g. around like 31 of `tests/Aspire.Dashboard.Tests/Integration/FrontendOpenIdConnectAuthTests.cs`) and then not need to add this testing-only parameter?

Reply:
Thank you, this was a very helpful suggestion. Removing test-only code is a nice improvement here.